core/src/commonMain/kotlin/com/ustadmobile/core/networkmanager/ImportJobRunner.kt

```diff
@@ -26,6 +26,10 @@ import kotlinx.atomicfu.atomic
 import kotlinx.coroutines.*
 import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
+import kotlinx.serialization.builtins.MapSerializer
```

Comment:
One issue on importjobrunner's function naming:

There is a function called 'startUpload'. That implies it is async and would continue in the background or something like that. In fact it actually does the whole upload. Can this be renamed to 'doUpload' or just 'upload'?